azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -113,10 +124,10 @@ private void sendSlaAlertEmail(final SlaOption slaOption, final String slaMessag
       if (!this.testMode) {
         try {
           message.sendEmail();
-          logger.info("Sent SLA email message " + slaMessage);
+          logger.info("Sent email message " + body);
```

Comment:
Would it be helpful to keep logging messages for sending sla email somewhere for debugging purpose?

Reply:
we can still grep "Sent email message" from the log to debug.

Reply:
As you mentioned, this would be a more general method used also in other places like the flow trigger. How could we distinguish whether the email message is for sla or flow trigger then?

Reply:
added logging message in sendSlaAlertEmail